### PR TITLE
Reviewer AMC: Set defaults if no config files are present

### PR DESCRIPTION
**Pull request by graemerobertson**
Fixes https://github.com/Metaswitch/chronos/issues/126.

Tested by running Chronos with no config files present, with just chronos.conf present and with both chronos.conf and chronos_cluster.conf present. Chronos started up with the correct options in each case.

### CODE REVIEW

File: src/main/globals.cpp [hunk 1]
```diff
@@ -86,30 +86,27 @@ Globals::~Globals()
...
   file.open(_cluster_config_file);
   if (file.is_open())
   {
     po::store(po::parse_config_file(file, _desc), conf_map);
-    po::notify(conf_map);
     file.close();
   }
 
   file.open(_config_file);
+  po::store(po::parse_config_file(file, _desc), conf_map);
```
Comment (bossmc) [inline src/main/globals.cpp]: What if `file` doesn't exist?

Comment (bossmc) [inline src/main/globals.cpp]:
```
file.open(_config_file);
if (file.open())
{
  po::store(po::parse_config_file(file, _desc), conf_map);
  file.close();
}
po::notify(conf_map);
```

**Comment by bossmc** [comment]
Fix in comments.

File: src/main/globals.cpp (hunk 1 above)
Comment (graemerobertson) [inline src/main/globals.cpp]: As discussed, this isn't right, and I've added a comment explaining why.

**Pull request merged**